base_synchro/wizard/base_synchro.py

```diff
@@ -48,18 +53,12 @@ def __init__(self, server):
     def get(self, ressource):
         return RPCProxyOne(self.server, ressource)
 
-class base_synchro(osv.TransientModel):
+class base_synchro(models.TransientModel):
     """Base Synchronization """
     _name = 'base.synchro'
 
-    _columns = {
-        'server_url': fields.many2one('base.synchro.server', "Server URL", required=True),
-        'user_id': fields.many2one('res.users', "Send Result To",),
-    }
-
-    _defaults = {
-        'user_id': lambda self, cr, uid, context: uid,
-    }
+    server_url = fields.Many2one('base.synchro.server', "Server URL", required=True)
+    user_id = fields.Many2one('res.users', "Send Result To",default=lambda self: self.env.user)
```

Comment:
Coding convention should be consistent.
Ex: 
1: Should have space after , 
2: Should have space before and after =